samplesheet/samplesheet.csv

```diff
@@ -1,3 +1,3 @@
-group,replicate,fastq_1,fastq_2,strandedness
-cage1,1,https://github.com/nf-core/test-datasets/raw/cageseq/testdata/cage1.fastq.gz,,forward
-cage2,1,https://github.com/nf-core/test-datasets/raw/cageseq/testdata/cage2.fastq.gz,,forward
\ No newline at end of file
+group,replicate,fastq_1,fastq_2
+cage1,1,https://github.com/nf-core/test-datasets/raw/cageseq/testdata/cage1.fastq.gz,
+cage2,1,https://github.com/nf-core/test-datasets/raw/cageseq/testdata/cage2.fastq.gz,
```

Comment:
```suggestion
cage1,1,https://github.com/nf-core/test-datasets/raw/cageseq/testdata/cage1.fastq.gz
cage2,1,https://github.com/nf-core/test-datasets/raw/cageseq/testdata/cage2.fastq.gz
```

Reply:
Sure we don't need the comma to indiate the additional column (read2)?

Reply:
oops, sorry, my bad. Forgot about that column 🙃